packages/admin/src/pages/scheme/[schemeId]/manage-editors/index.page.tsx

```diff
@@ -74,7 +80,7 @@ const ManageEditors = ({
           <div className="govuk-button-group">
             <CustomLink
               isButton
-              href={`/scheme/${schemeId}/manage-editors/add`}
+              href={`/scheme/${schemeId}/manage-editors/add-editor?schemeName=${schemeName}`}
```

Comment:
We can use the `schemeId` in the url to get the schemeName rather than relying on this query string

Reply:
I don't personally think it's worth another service call to get information the previous page has access too already. 
If this were an applicant facing page I'd agree